migrations/20150809023413-add-stats-time.js

```diff
@@ -1,15 +0,0 @@
-var async = require('async');
```

Comment:
@justinbmeyer oh and most importent thats why we have automated tests :) they all pass :)